src/pymovements/gaze/integration.py

```diff
@@ -234,26 +234,26 @@ def from_numpy(
 
     pixel_columns = None
     if pixel is not None:
-        df = pl.from_numpy(data=pixel, orient=orient).select(pl.all().prefix('pixel_'))
+        df = pl.from_numpy(data=pixel, orient=orient).select(pl.all().name.prefix('pixel_'))
         dfs.append(df)
         pixel_columns = df.columns
 
     position_columns = None
     if position is not None:
-        df = pl.from_numpy(data=position, orient=orient).select(pl.all().prefix('position_'))
+        df = pl.from_numpy(data=position, orient=orient).select(pl.all().name.prefix('osition_'))
```

Comment:
I've had this mistake while renaming -- why isn't this tested?

Reply:
this is actually hard to test because the column names are just internal (i.e. we would need to mock).
at the end of the function we initialize a GazeDataFrame with `position_columns=position_columns`, which has the effect that the columns are nested into `position` and the original columns are removed, i.e. the original column names don't matter much.